packages/synapse-react-client/src/components/FeaturedDataTabs/FacetPlotsCard.tsx

```diff
@@ -134,124 +133,131 @@ const FacetPlotsCard: React.FunctionComponent<FacetPlotsCardProps> = ({
-          {description && (
-            <Typography variant={'body1'} sx={{ color: 'grey.700', my: 2 }}>
-              <ShowMore summary={description} maxCharacterCount={200} />
-            </Typography>
-          )}
+    // If we are showing a facet selection based card, then set the selectedFacetValue.  For example, facet column "study" with value "ROSMAP"
+    const selectedFacet: FacetColumnResultValueCount | undefined = queryMetadata
+      ?.facets!.map(item => {
+        const facetValues: FacetColumnResultValueCount[] = (
+          item as FacetColumnResultValues
+        ).facetValues
+        if (facetValues) {
+          const filteredFacetValues: FacetColumnResultValueCount[] =
+            facetValues.filter(facetValue => {
+              return facetValue.isSelected
+            })
+          return filteredFacetValues.length > 0
+            ? filteredFacetValues[0]
+            : undefined
+        } else {
+          return undefined
+        }
+      })
+      .filter(x => x !== undefined)[0]
 
-          {detailsPagePath && selectedFacetValue && (
-            <Box sx={{ my: 2 }}>
-              <Button
-                variant={'contained'}
-                href={detailsPagePath}
-                color={'secondary'}
-              >
-                Explore {selectedFacetValue}
-              </Button>
-            </Box>
-          )}
-        </FacetPlotsCardTitleContainer>
+    if (selectedFacet && selectedFacet.value) {
+      return selectedFacet?.value
+    }
+    return ''
+  }, [facetsToPlot, queryMetadata?.facets])
 
-        {/* create a plot for every facet to be plotted */}
-        {facetPlotDataArray.map((plotData, index) => {
-          return (
-            <React.Fragment key={index}>
-              <FacetPlotsCardPlotContainer
-                className={FACET_PLOTS_CARD_PLOT_CONTAINER_CLASSNAME}
-                sx={{
-                  pt: index === 0 ? 5 : 0,
-                  gridRow: `plot${index}`,
-                }}
-                key={index}
-              >
-                {index != 0 && <Divider sx={{ mt: 2, mb: 4 }} />}
-                <Box sx={{ minHeight: '130px' }}>
-                  <SizeMe monitorHeight noPlaceholder>
-                    {({ size }) => (
-                      <Box
-                        sx={{
-                          display: 'flex',
-                          justifyContent: 'center',
-                          alignItems: 'center',
-                        }}
-                      >
-                        <Plot
-                          key={`${facetsToPlot![index]}-${size.width!}`}
-                          layout={layout}
-                          data={plotData?.data ?? []}
-                          style={getPlotStyle(size.width, 'PIE', 150)}
-                          config={{ displayModeBar: false }}
-                        />
-                      </Box>
+  const isShowingMultiplePlots = facetPlotDataArray.length > 1
```

Comment:
Bad diff, no changes from here through the rest of the component. By removing the loading case, these lines all have one fewer level of indentation.

Reply:
Reminded me to ignore whitespace.  Much better :D